app/assets/javascripts/discourse/templates/components/topic-join-group-notice.hbs

```diff
@@ -0,0 +1,6 @@
+{{accessViaGroupText}}
+<br>
```

Comment:
I prefer to avoid `<br>` tags for formatting. Instead, I'd suggest to wrap the text in a span or div and add a bottom margin.